.github/workflows/tox.yml

```diff
@@ -1,95 +0,0 @@
-name: Tox
-
-on:
-  push:
-    branches:
-    - functional
-  pull_request:
-    branches:
-    - functional
-
-jobs:
-  tox-tests:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.10"]
-        tox-env-factors: ["base", "atlas"]
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Install boost
-      run: |
-        wget https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.gz
-        echo 7bd7ddceec1a1dfdcbdb3e609b60d01739c38390a5f956385a12f3122049f0ca boost_1_76_0.tar.gz > boost_hash.txt
-        sha256sum -c boost_hash.txt
-        tar xzf boost_1_76_0.tar.gz
-        mkdir -p boost/include
-        mv boost_1_76_0/boost boost/include/
-        echo "BOOST_ROOT=${PWD}/boost" >> $GITHUB_ENV
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install python dependencies
-      run: |
-        python -m pip install --upgrade pip setuptools wheel
-    - name: Install C++ dependencies
-      run: |
-        sudo apt install libboost-all-dev
-    - name: Test with tox
-      run: |
-        pyversion_no_dot="${{ matrix.python-version }}"
-        pyversion_no_dot="${pyversion_no_dot/./}"
-        tox_env_factor="${{ matrix.tox-env-factors }}"
-        pip install tox clang-format
-        tox -r -e py${pyversion_no_dot}-${tox_env_factor}
-    - name: Archive code coverage results
-      uses: actions/upload-artifact@v2
-      with:
-        name: code-coverage-report
-        path: tests/_reports/
-
-  tox-quickstart-guide:
```

Comment:
It looks like the quickstart-guide related actions haven't been moved to another file but just removed. I would keep them.

Reply:
I suspect that was added to the CI on the main branch after I created the PR :) I added them to the current workflows.